modules/administration/pages/monitoring.adoc

```diff
@@ -145,136 +76,150 @@ zypper in golang-github-prometheus-prometheus
 
-. In the {productname} {webui}, navigate to menu:Admin[Manager Configuration > Monitoring].
-. Click btn:[Enable services].
+== Configure {productname} Monitoring
 
-image::monitoring_enable_services.png[scaledwidth=80%]
+=== Server Self Monitoring
 
+With {productname}{nbsp}4, you can enable the Server to expose Prometheus self-health metrics. Those metrics are made available by instrumentation of the Java application combined with Prometheus exporters. Together, they provide metrics about the underlying Hardware, Operating System and {productname} internals.
 
-.Procedure: Configuring Monitoring Formulas
+These exporter packages are shipped with {productname} Server:
 
-. In the {susemgr} {webui}, open the details page of the system to be monitored, and navigate to the [guimenu]``Formulas`` tab.
-. Check the [guimenu]``Monitoring`` checkbox to select all monitoring formulas, and click btn:[Save].
-. Apply the highstate.
+* Node exporter: [systemitem]``golang-github-prometheus-node_exporter``.
+See https://github.com/prometheus/node_exporter.
+* PostgreSQL exporter: [systemitem]``golang-github-wrouesnel-postgres_exporter``.
+See https://github.com/wrouesnel/postgres_exporter.
+* JMX exporter: [systemitem]``prometheus-jmx_exporter``.
+See https://github.com/prometheus/jmx_exporter.
+* Apache exporter: [systemitem]``golang-github-lusitaniae-apache_exporter``.
+See https://github.com/Lusitaniae/apache_exporter.
 
+These exporter packages are shipped with {productname} Proxy:
 
-.Procedure: Configuring the Exporters
+* Node exporter: [systemitem]``golang-github-prometheus-node_exporter``.
+See https://github.com/prometheus/node_exporter.
+* Squid exporter: [systemitem]``golang-github-boynux-squid_exporter``.
+See https://github.com/boynux/squid-exporter.
 
-. In the {susemgr} {webui}, open the details page of the system to be monitored, and navigate to the menu:Formulas[Prometheus Exporters] tab.
-. Check the [guimenu]``Enabled`` checkbox for both the Node and the Postgres Exporter.
-. In the [guimenu]``Postgres Exporter`` section, in the [guimenu]``Data Source Namer`` field, enter the path to your data source (for example, [systemitem]``postgresql://user:passwd@localhost:5432/database?sslmode=disable``).
-. Click btn:[Save Formula].
-. Apply the highstate.
+The packages above are pre-installed in {productname} Server and Proxy, but their respective systemd daemons are disabled by default. 
 
-image::monitoring_configure_formula.png[scaledwidth=80%]
+.Procedure: Enabling Self Monitoring for {productname}
 
+. In the {productname} {webui}, navigate to menu:Admin[Manager Configuration > Monitoring].
+. Click btn:[Enable services].
+. Restart Tomcat and Taskomatic.
+. Navigate to the URL of your Prometheus server, on port 9090 (for example, [literal]``http://example.com:9090``)
+. In the Prometheus UI, navigate to menu:[Status > Targets] and confirm that all the endpoints on the ``mgr-server`` group are up.
```

Comment:
```suggestion
. In the Prometheus UI, navigate to menu:[Status > Targets] and confirm that all the endpoints on the ``mgr-server`` group are active.
```
??

Reply:
'Active' does not sound correct. Prometheus sorts the endpoints as 'Up' or 'Down' (and displays them like that in the UI). There is no concept of 'active'.

Reply:
If `up` is used in the interface, then stick with it.